.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea/24-jan-honey-badger-brigade-blogging-site.iml
```

Comment:
You can just put `.idea/` in here to ignore everything in it.

However, ignoring files that have already been tracked won't actually work. You'll need to run the following commands to remove .idea from the repo:

`git rm -r --cached .idea/` - This will remove the files from git
`git add .` - To add the removed files (make sure you do a git status first so you don't add random crap)
`git commit -m "Removing ignored files"`
`git push origin S1`